elastalert/test_rule.py

```diff
@@ -79,44 +81,44 @@ def test_file(self, args):
         try:
             res = es_client.count(index, doc_type=doc_type, body=count_query, ignore_unavailable=True)
         except Exception as e:
-            logging.info("Error querying Elasticsearch:")
-            logging.info(repr(e)[:2048])
+            print("Error querying Elasticsearch:", file=sys.stderr)
+            print(repr(e)[:2048], file=sys.stderr)
             return None
 
         num_hits = res['count']
-        logging.info("Got %s hits from the last %s day%s" % (num_hits, args.days, 's' if args.days > 1 else ''))
+        print("Got %s hits from the last %s day%s" % (num_hits, args.days, 's' if args.days > 1 else ''))
         print_terms(terms, '')
 
         # Check for missing keys
         pk = conf.get('primary_key')
         ck = conf.get('compare_key')
         if pk and not lookup_es_key(terms, pk):
-            logging.info("Warning: primary key %s is either missing or null!")
+            print("Warning: primary key %s is either missing or null!", file=sys.stderr)
         if ck and not lookup_es_key(terms, ck):
-            logging.info("Warning: compare key %s is either missing or null!")
+            print("Warning: compare key %s is either missing or null!", file=sys.stderr)
 
         include = conf.get('include')
         if include:
             for term in include:
                 if not lookup_es_key(terms, term) and '*' not in term:
-                    logging.info("Included term %s may be missing or null" % (term))
+                    print("Included term %s may be missing or null" % (term), file=sys.stderr)
 
         for term in conf.get('top_count_keys', []):
             # If the index starts with 'logstash', fields with .raw will be available but won't in _source
             if term not in terms and not (term.endswith('.raw') and term[:-4] in terms and index.startswith('logstash')):
-                logging.info("top_count_key %s may be missing" % (term))
-        logging.info('')
+                print("top_count_key %s may be missing" % (term), file=sys.stderr)
```

Comment:
These technically work, though I'm curious why you didn't instead opt for [logging.error](https://docs.python.org/2/library/logging.html#logging.error), then direct ERROR and CRITICAL to stderr.